wlutil/br/br.py

```diff
@@ -37,6 +38,48 @@
 
 exit""")
 
+def buildConfig():
+    """Construct the final buildroot configuration for this environment. After
+    calling this, it is safe to call 'make' in the buildroot directory."""
+
+    # Major/minor version of the linux kernel headers included with our
+    # toolchain. This is not necessarily the same as the linux kernel used by
+    # Marshal, but is assumed to be <= to the version actually used.
+    linuxHeaderVer = sp.run(
+      "{ echo '#include <linux/version.h>' ; echo 'LINUX_VERSION_CODE' ; } | riscv64-unknown-linux-gnu-gcc -E -xc - | tail -n 1",
```

Comment:
It's possible to just exec `riscv64-unknown-linux-gnu-gcc` directly instead of using the shell.

Untested:
```python
linuxHeaderTest = """#include <linux/version.h>
LINUX_VERSION_CODE
"""
linuxHeaderVer = sp.run(['riscv64-unknown-linux-gnu-gcc', '-E', '-xc', '-'],
  input=linuxHeaderTest, stdout=sp.PIPE, universal_newlines=True)
linuxHeaderVer = linuxHeaderVer.stdout.readlines()[-1].strip()
```
There's probably a more Pythonic way to get the last line of stdout.

Reply:
done